util/test/perf/perfgraph.js

```diff
@@ -393,6 +404,22 @@ function setupScreenshotToggle(g, graphInfo, screenshotToggle) {
 }
 
 
+// Setup the close graph button
+function setupCloseGraphToggle(g, graphInfo, closeGraphToggle) {
+  closeGraphToggle.style.visibility = 'visible';
+
+  closeGraphToggle.onclick = function() {
+    var checkBox = getCheckboxForGraph(g);
+    checkBox.checked = false;
+
+    // TODO instead of completely redrawing with displaySelectedGraphs() we
+    // could just remove the divs associated with the graph being removed to
+    // speed up this operation and make it less "jittery"
+    displaySelectedGraphs();
```

Comment:
Eh, I don't feel the need to update the comment for that. That limitation of "Display" is documented elsewhere